classes.py

```diff
@@ -0,0 +1,60 @@
+
+import os
+from os.path import join
+
+from mutagen.easyid3 import EasyID3
+
+
+class MusicFile:
+    def __init__(self, file_path):
+        self.file_path = file_path
+
+        self.tracknumber = None
+        self.title = None
+        self.artist = None
+        self.album_artist = None
+        self.album_title = None
+        self.year = None
+
+        self.read_tags()
+        print(self.album_title, self.tracknumber, self.title)
+
+    @classmethod
+    def is_music_file(cls, file_path):
+        return file_path.endswith(".mp3")
+
+    def read_tags(self):
+        audio = EasyID3(self.file_path)
+        try:
+            self.tracknumber = int(audio["tracknumber"][0])  # TODO: konwertuj do int
+        except ValueError:
+            pass
```

Comment:
Uciszyłaś problem, zamiast go rozwiązać. Zwróć uwagę, że w pliku jest informacja o numerze ścieżki - tylko jest w innym formacie niż tego oczekujesz. Gdyby ta wartość była równa `"10"`, to twój program by sobie z tym poradził, ale jest równa `"10/25"` (10-ta ścieżka, z 25-ciu na płycie).
Spróbuj wyciągnąć ten właściwy numer z `audio["tracknumber"][0]`.

Reply:
A teraz?

Reply:
Jeśli program się nie wywala, to na razie może być.
Ale odczytywanie tych tagów, to pewnie będzie kolejny issue.